user/deployment/npm.md

```diff
@@ -23,14 +21,27 @@ deploy:
   api_key: "YOUR_AUTH_TOKEN"
 ```
 
-Always [encrypt](/user/encryption-keys/#Usage) your API key. 
+You can have the `travis` tool set up everything for you:
+
+```bash
+$ travis setup npm
+```
+
+Keep in mind that the above command has to run in your project directory, so
+it can modify the `.travis.yml` for you.
 
-## NPM Auth Token
+## NPM auth token
 
 Your NPM Auth Token can be obtained by:
 
-1. Login into [npmjs.com](http://www.npmjs.com), you [generating a new token](https://www.npmjs.com/settings/tokens).
-2. Login via the CLI with [npm adduser](https://docs.npmjs.com/cli/adduser) and look at the `authToken` value in your `~/.npmrc` file.
+1. Login to your NPM account, and [generating a new token](https://www.npmjs.com/settings/tokens).
```

Comment:
This should probably be "Log in" . Obviously the mistake predates your PR, but we may as well fix.